products/caregivers/Improve Facility Selection/10-10 Health Apps-10-10CG-[date].md

```diff
@@ -2,71 +2,97 @@
+  - Are you deprecating or removing any APIs?
+    - No
+  - Do you have API documentation?
+    - No, the only change is a new endpoint in the caregivers controller.
+  - Describe expected call patterns
+    - `vets-website` calls this endpoint to a list of paginated facilities
 - External API changes
-    + List new or modified APIs for upstream or external systems
-    + Describe expected call patterns
-    + What PII or PHI will be transmitted to/from the external systems?
+  - List new or modified APIs for upstream or external systems
+    - No Changes
+  - Describe expected call patterns
+    - No Changes
+  - What PII or PHI will be transmitted to/from the external systems?
+    - No Changes
+  - List any required background processing
+    - No Changes
+  - Describe error and dead letter handling
+    - No Changes
 - Background jobs
-    + List any required background processing
-    + Describe error and dead letter handling
+  - List any required background processing
+  - Describe error and dead letter handling
 - Data storage
-    + Describe new or modified databases, tables or columns
-    + Describe indexes and constraints
-    + Identify PII and PHI and where and how it will be stored, processed, expired and deleted
+  - Describe new or modified databases, tables or columns
+    - No Changes. The facility that is ultimately returned is used to pass the facility id as part of the 1010CG Form submission.
+  - Describe indexes and constraints
+    - No Changes
+  - Identify PII and PHI and where and how it will be stored, processed, expired and deleted
+    - No Changes
 - Libraries and dependencies
-    + List new or updated dependencies
+  - List new or updated dependencies
 - Metrics, logging, observability, alerting
-    + Identify key areas to monitor
+  - Identify key areas to monitor
 - Infrastructure and network changes
-    + List any changes or additions
+  - List any changes or additions
 - Test strategy
-    + Describe automated, manual and user acceptance test strategy
-    + Describe required test data and test user accounts
-         + This is an aunauthenticated form, meaning that a user does not need to be signed in to complete and submit it.  We do not use any data from an authenticated user to pre-fill this form or provide any statuses.
+  - Describe automated, manual and user acceptance test strategy
+  - Describe required test data and test user accounts
+    - This is an aunauthenticated form, meaning that a user does not need to be signed in to complete and submit it. We do not use any data from an authenticated user to pre-fill this form or provide any statuses.
 - Rollout plan
-    + List scope of any feature flags
-         + Feature flag: caregiver_use_facilities_API
-         + Feture flag description: Allow list of caregiver facilites to be fetched by way of the Facilities API
-    + Identify other teams to coordinate with
```

Comment:
- There are no other teams that are dependent on our change.  We will coordinate with the Caregiver stakeholders to ensure the support staff are prepared for the change in how the form will handle facility selection, however there are no changes to the output they will receive downstream.